docs/MANUAL.md

```diff
@@ -150,6 +150,65 @@ kodelet conversation delete <conversation-id>
 kodelet conversation delete --no-confirm <conversation-id>
 ```
 
+## Shell Completion
+
+Kodelet provides shell completion support for bash, zsh, and fish. This enables tab completion for commands and flags, making the CLI experience more efficient.
+
+### Setup Instructions
+
+**Bash:**
+
+To load completions in your current shell session:
+```bash
+source <(kodelet completion bash)
```

Comment:
no need to mention how to load in the current shell session, to keep it simple just mention how to load completions for every new session -  `echo 'source <(kodelet completion bash)' >> ~/.bashrc` is fine